🐛 Minor fixes

bisgaard-itis opened the pull request:
<!-- Common title prefixes/annotations:
PREFIX:

  WIP: work in progress
  🐛    Fix a bug.
  ✨    Introduce new features.
  ♻️     Refactor code.
  🚑️    Critical hotfix.
  ⚗️     Perform experiments.
  ⬆️    Upgrade dependencies.
  📝    Add or update documentation.
  🗑️    Deprecate code that needs to be cleaned up.
  ⚰️     Remove dead code.
  🔥    Remove code or files.
  🔨    Add or update development scripts.

or from https://gitmoji.dev/
-->

## What do these changes do?
Further minor changes to make e2e tests happy
<!-- Explain REVIEWERS what is this PR about -->


## Related issue/s

<!-- Enumerate REVIEWERS other issues

e.g.

- #26 : node_ports should have retry policies when upload/download fails  (FIXED)
- ITISFoundation/osparc-issues#304: (Part 2) Prep2Go: creating features to support complex S4L scripts (IMPLEMENTED)

-->


## How to test

<!-- Give REVIEWERS some hits or code snippets on how could this be tested -->

pcrespov commented on clients/python/client/osparc/_files_api.py:
```diff
@@ -91,10 +91,10 @@ async def upload_file_async(self, file: Union[str, Path]) -> File:
             request_type="post",
             url=links.abort_upload,
             auth=self._auth,
+            follow_redirects=True,
```
for the pre-signed links, right?

bisgaard-itis replied:
This was because the `abort` and `complete` links I got back from the Api server were using http instead of https. This is very strange, because they are generated using starlette.Request. But in any case I will switch to using relative links soon (as discussed) so this shouldn't be an issue

pcrespov commented on clients/python/client/osparc/_http_client.py:
```diff
@@ -8,16 +8,9 @@ class AsyncHttpClient:
-        request_type: Optional[str] = None,
-        url: Optional[str] = None,
-        auth: Optional[httpx.BasicAuth] = None,
+        self, *, request_type: Optional[str] = None, url: Optional[str] = None, **kwargs
```
Q: the client needs to work for 3.6, right?  we should add at some point `py-upgrade` pre-commit hooks

pcrespov replied:
MINOR: I would rename `kwargs` by `httpx_async_client_kwargs`

bisgaard-itis replied:
good point, I have fixed this

bisgaard-itis commented:
@pcrespov FYI I found out why my test wasn't working on master and it had to do with the fact that I was (mistakingly) sending the credentials to s3 when uploading the file chunks. I have fixed this now so the e2e tests should work as soon as this gets in and https://git.speag.com/oSparc/e2e-portal-testing/-/merge_requests/19 gets in. I also introduced some logging (just using print statements for now). I will have to introduce a proper logging system at some point.
